provider/cmd/pulumi-resource-eks/schema.json

```diff
@@ -268,6 +268,11 @@
                     "type": "boolean",
                     "description": "Use the latest recommended EKS Optimized Linux AMI with GPU support for the worker nodes from the AWS Systems Manager Parameter Store.\n\nDefaults to false.\n\nNote: `gpu` and `amiId` are mutually exclusive.\n\nSee for more details:\n- https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html\n- https://docs.aws.amazon.com/eks/latest/userguide/retrieve-ami-id.html"
                 },
+                "ignoreScalingChanges": {
```

Comment:
This seemed to have been overwritten during a merge of one of the recent changes we've merged in. This brings it back into the schema.

Reply:
It was introduced here: https://github.com/pulumi/pulumi-eks/pull/1380